### PR TITLE
Change pate-werden route to masifunde-gemeinschaft

### PR DESCRIPTION
**Pull request by sarunast**
Change url `/wie-sie-helfen/pate-werden` to `/wie-sie-helfen/masifunde-gemeinschaft`

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit 4e72162f060b494918c428a74b8dddc21ba6cd30

https://deploy-preview-231--masifunde.netlify.com

File: routes.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ addRoute('/wer-wir-sind', WhoWeAre)
 addRoute('/wer-wir-sind/team-de', TeamDE)
 addRoute('/wer-wir-sind/team-sa', TeamSA)
 addRoute('/wie-sie-helfen', HowToSupport)
-addRoute('/wie-sie-helfen/pate-werden', BecomeSponsor)
+addRoute('/wie-sie-helfen/patenprogramm', BecomeSponsor)
```
Comment (erikjalevik) [inline routes.js]: Umm no, it should be "masifunde-gemeinschaft" :)

Comment (erikjalevik) [reply routes.js]: And what about the English version? I'd suggest "masifunde-community" for that.

Comment (sarunast) [reply routes.js]: So it should be `/wie-sie-helfen/masifunde-gemeinschaf` ?

I don't think we have English versions for these URL's. Only the blog has some kind of slug system as long as I remember.

Comment (erikjalevik) [reply routes.js]: Hehe, noooooo!

`/wie-sie-helfen/masifunde-gemeinschaft` 

:D

Comment (erikjalevik) [reply routes.js]: And for English, OK, then we're good.

Comment (sarunast) [reply routes.js]: Oki doki updated. No angry 🐑 ...

**Comment by sarunast** [comment]
Now that I think of it for SEO we should add redirect to that page @erikjalevik.

**Comment by sarunast** [comment]
Included the redirecting from old page to new page as well.

**Pull request merged**